README.md

```diff
@@ -1,110 +1,124 @@
 # The missing introduction to APCA
 
-## What is APCA?
+## _What is APCA in three sentences:_
```

Comment:
Headings should not be underlined and should not end with a color

Reply:
That is not underlining, it is italicizing